results/cmd/api/main.go

```diff
@@ -57,7 +59,24 @@ type server struct {
 	db *sql.DB
 }
 
-func (s *server) CreateTaskRun(ctx context.Context, req *pb.CreateTaskRunRequest) (*pb.TaskRun, error) {
-	// TODO: implement this.
-	return nil, status.Errorf(codes.Unimplemented, "method CreateTaskRun not implemented")
+// Receive TaskRun from Watcher and save it to local Sqlite Server
+func (s *server) CreateTaskRun(ctx context.Context, req *pb.CreateTaskRunRequest) (*pb.CreateTaskRunResponse, error) {
+	database := s.db
+	statement, _ := database.Prepare("CREATE TABLE IF NOT EXISTS taskrun (logid INTEGER PRIMARY KEY, taskrunlog BLOB)")
```

Comment:
I think we should avoid creating the table at runtime, and just throw an error if the table doesn't exist. Separately, we can and should define a schema with instructions and/or script for how to create the table.

Reply:
Okay, I will move it to main function and add some error handling. 
What does the schema with instructions mean?